cmd/kubermatic-installer/cmd_deploy.go

```diff
@@ -186,6 +190,11 @@ func DeployFunc(logger *logrus.Logger, versions kubermaticversion.Versions, opt
 			)
 		}
 
+		// default App Catalog feature is only available for EE, so error out early if being used with CE
```

Comment:
We use compile-time settings, never runtime checks for the edition.

Reply:
ok. I guess I then the only way I could think of is to create a stack_ce.go and a stack_ee.go which then get called in [pkg/install/stack/kubermatic-master/stack.go](https://github.com/kubermatic/kubermatic/pull/12623/files#diff-eea3bf1b32bfc7d0d71cba02976c90303e11a71e008542896f99fa4ffdfc2466). The stack_ce version is essentially a no-op. Would this be our preferred approach?

Reply:
Yep, exactly. Similar to https://github.com/kubermatic/kubermatic/blob/main/cmd/master-controller-manager/wrappers_ce.go#L33-L40